modules/kubernetes/kube2iam/templates/values.yaml.tpl

```diff
@@ -0,0 +1,38 @@
+extraArgs:
+  namespace-restrictions: true
+
+#   base-role-arn: arn:aws:iam::0123456789:role/
+#   default-role: kube2iam-default
+#   api-server: ...
+#   api-token: ...
+
+host:
+  ip: $(HOST_IP)
+  # https://github.com/jtblin/kube2iam#iptables
+  iptables: true
+  # Use calico
+  interface: cali+
+  port: 8181
+
+prometheus:
+  # Port to expose the /metrics endpoint on. If unset, defaults to `host.port`
+  # metricsPort: 9543
+  service:
+    enabled: true
+
+  serviceMonitor:
+    # Create prometheus-operator ServiceMonitor
+    enabled: true
+    # Labels to add to the service monitor
+    labels:
+      xkf.xenit.io/monitoring: platform
+
+priorityClassName: "platform-high"
+
+rbac:
+  ## If true, create & use RBAC resources
+  ##
+  create: true
```

Comment:
is this the same as the default value? If so it can be removed

Reply:
psp is deafult but not rbac.